Assets/MRTK/Providers/Windows/WindowsDictationInputProvider.cs

```diff
@@ -312,6 +318,8 @@ public override void Update()
         /// <inheritdoc />
         public override async void Disable()
         {
+            base.Disable();
```

Comment:
The more I see all these `base` classes added, the more wary I'm feeling about this change...if these weren't required before (and the MRTK classes weren't doing it well!), we likely have custom services out there that don't call these base classes, which might now become broken if we rely on the base calls for behavior and state.

Reply:
interesting point. any thoughts on how to make it more robust?

Reply:
implemented the backing bool as a nullable. added an assert to check for null